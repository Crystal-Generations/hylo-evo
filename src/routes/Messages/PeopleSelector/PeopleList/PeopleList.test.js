import { shallow } from 'enzyme'
import React from 'react'

import PeopleList from './PeopleList'

it('does something', () => {
  const people = [
    {
      id: '1',
      name: 'Wombat',
      avatarUrl: 'https://flargle.com',
      community: 'Wombats'
    },
    {
      id: '2',
      name: 'Aardvark',
      avatarUrl: 'https://argle.com',
      community: 'Aardvarks'
    },
    {
      id: '3',
      name: 'Ocelot',
      avatarUrl: 'https://wargle.com',
      community: 'Ocelots'
    }
  ]
  const wrapper = shallow(<PeopleList people={people} />)
  expect(wrapper).toMatchSnapshot()
})
