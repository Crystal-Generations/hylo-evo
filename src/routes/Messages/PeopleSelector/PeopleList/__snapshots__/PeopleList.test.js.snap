// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does something 1`] = `
<div
  data-stylename="people-list-container"
>
  <div>
    <h2
      data-stylename="people-list-header"
    >
      All Contacts
    </h2>
    <ul
      data-stylename="people-list"
    >
      <PeopleListItem
        key="1"
        onClick={[Function]}
        person={
          Object {
            "avatarUrl": "https://flargle.com",
            "community": "Wombats",
            "id": "1",
            "name": "Wombat",
          }
        }
      />
      <PeopleListItem
        key="2"
        onClick={[Function]}
        person={
          Object {
            "avatarUrl": "https://argle.com",
            "community": "Aardvarks",
            "id": "2",
            "name": "Aardvark",
          }
        }
      />
      <PeopleListItem
        key="3"
        onClick={[Function]}
        person={
          Object {
            "avatarUrl": "https://wargle.com",
            "community": "Ocelots",
            "id": "3",
            "name": "Ocelot",
          }
        }
      />
    </ul>
  </div>
</div>
`;
